routes/web.php

```diff
@@ -250,19 +250,33 @@
             Route::get('/archive/{id}', 'AdminDash@archiveIncident');
             Route::get('/delete/{id}', 'AdminDash@deleteIncident');
         });
-        Route::prefix('realops')->middleware('toggle:realops')->middleware('permission:staff')->group(function () {
-            Route::get('/', 'RealopsController@adminIndex');
-            Route::get('/export', 'RealopsController@exportData');
-            Route::get('/create', 'RealopsController@showCreateFlight');
-            Route::post('/create', 'RealopsController@createFlight');
-            Route::post('/create/bulk', 'RealopsController@bulkUploadFlights');
-            Route::get('/edit/{id}', 'RealopsController@showEditFlight');
-            Route::put('/edit/{id}', 'RealopsController@editFlight');
-            Route::get('/{id}', 'RealopsController@deleteFlight');
-            Route::put('/assign-pilot/{id}', 'RealopsController@assignPilotToFlight');
-            Route::get('/remove-pilot/{id}', 'RealopsController@removePilotFromFlight');
-            Route::get('/delete/{id}', 'RealopsController@deleteFlight');
-            Route::post('/dump-data', 'RealopsController@dumpData');
+        Route::prefix('realops')->middleware('toggle:realops')->group(function () {
```

Comment:
I don't know if this is the best way to do this... looks really messy. @iccowan do you have any suggestions here?

Reply:
No, but I hate to add another view that is identical. We already have a lot of those throughout the site and I'm working on eliminating the duplicative views as I update things. Making duplicate views is a real pain to maintain. I'm hoping that @iccowan knows a better way to do this routing so it isn't so convoluted.

Reply:
I would probably do this with 2 groups. Have the first group with all 3 selections applied (you should be able to combine "hasRole" and "can"). Then the second group, have just the routes that you want staff and events to have. That should solve it and remove the duplicate code.